src/js/toggler.js

```diff
@@ -1,42 +1,21 @@
-import $ from 'jquery'
+document.addEventListener('DOMContentLoaded', () => {
+  document.querySelectorAll('.js-group').forEach(group => {
```

Comment:
I'd cache the QSA calls personally to a var.

Reply:
It's only used once, that's why I didn't do that here. @Johann-S, what do you think?

Reply:
I think we should create a var because we have to convert the result of `document.querySelectorAll` into a real array with:
```
const jsGroupList = [].slice.call(document.querySelectorAll('.js-group'))
```

Because we cannot use `.forEach` on the result of `document.querySelectorAll` on Edge